R/utilities.R

```diff
@@ -1175,89 +1214,100 @@ split_SummarizedExperiment_by_feature_to_list = function(.data){
 #' @param name A character name of the attribute
 #'
 #' @return A tibble with an additional attribute
-add_attr = function(var, attribute, name) {
-  attr(var, name) <- attribute
-  var
+add_attr <- function(var, attribute, name) {
+    attr(var, name) <- attribute
+    var
 }
 
-is_filer_columns_in_column_selection = function(.data, ...){
-  # columns = enquos(columns)
-  tryCatch({
-    .data |>
-      slice(0) |>
-      dplyr::filter(...)
-    TRUE
-  },
-  error = function(e) FALSE)
+is_filer_columns_in_column_selection <- function(.data, ...) {
+    # columns = enquos(columns)
+    tryCatch({
+        .data |>
+            slice(0) |>
+            dplyr::filter(...)
+        TRUE
+    },
+    error = function(e) FALSE)
 }
 
-check_if_assays_are_NOT_consistently_ordered = function(se){
-  
-  # If I have any assay at all
-  assays(se) |> length() |> gt(0) &&
+check_if_assays_are_NOT_consistently_ordered <- function(se) {
     
-    # If I have more than one assay with colnames
-    Filter(
-      Negate(is.null),
-      assays(se, withDimnames = FALSE) |>  
-        as.list() |> 
-        map(colnames)
-    ) |> 
-    length() |>
-    gt(0) &&
-    
-  # If I have lack of consistency
-  se |> 
-    assays(withDimnames = FALSE) |>
-    as.list() |>
-    purrr::map_dfr(colnames) |>
-    apply(1, function(x) x |> unique() |> length()) |>
-    equals(1) |>
-    all() |>  
-    not()
+    # If I have any assay at all
+    assays(se) |> length() |> gt(0) &&
+        
+        # If I have more than one assay with colnames
+        Filter(
+            Negate(is.null),
+            assays(se, withDimnames = FALSE) |>  
+                as.list() |> 
+                map(colnames)
+        ) |> 
+        length() |>
+        gt(0) &&
+        
+        # If I have lack of consistency
+        se |> 
+        assays(withDimnames = FALSE) |>
+        as.list() |>
+        purrr::map_dfr(colnames) |>
+        apply(1, function(x) x |> unique() |> length()) |>
+        equals(1) |>
+        all() |>  
+        not()
 }
 
-check_if_assays_are_NOT_overlapped = function(se){
-  
-  # If I have any assay at all
-  assays(se) |> length() |> gt(0) &&
+check_if_assays_are_NOT_overlapped <- function(se, dim = "cols") {
+    stopifnot(dim %in% c("rows", "cols"))
+    if (dim == "rows") {
+        namefcn <- rownames
+        dimfcn <- nrow
+    } else {
+        namefcn <- colnames
+        dimfcn <- ncol
```

Comment:
I try to keep the codebase acronyms and abbreviations free. Would you mind replacing `...fcn` with a verbose, self-explanatory alternative?

This helps all developers involved immediately understand.